pipelines/internal/commands/run/run.go

```diff
@@ -385,12 +381,26 @@ func buildRequest(filename, project string) (*genomics.RunPipelineRequest, error
 		})
 	}
 
+	resources := &genomics.Resources{
+		ProjectId:      project,
+		VirtualMachine: vm,
+	}
+	if *regions != "" {
```

Comment:
This silently makes regions have higher precedence than zones... It's maybe OK, since we default to a zone, and therefore if a user specifies --regions=X they will get the behaviour they want, but it is surprising.

[I regret not making the API take 'locations' instead of zones and regions, but here we are.]

There are a few options:
(1) leave as is.  That's not great.
(2) drop the else.  This will mean when someone says '--regions=X' it will fail the request.  That's bad.
(3) [my favorite] drop the default zone from the flags and add it later, if the set of regions and zones are empty.  This has the downside of being a hidden default, so to speak, but produces the best user experience I think.

Thoughts?

Reply:
So functionality wise 1 and 3 seem to be the same, with one benefit that 3 allows us to catch cases when people supply both - not sure how useful that is, but it could remove the confusion for some edge cases.

Implemented 3 to default later, and to throw an exception if both flags have been supplied.

Reply:
You don't strictly need the check for both: the request will fail at the API level

I'd imagined something like:

```
if regions != "" {
  ...
}
if zones != "" {
  ...
}
if len(resources.Zones) + len(resources.Regions) == 0 {
  resources.Zones = []string{"us-east1d"}
}
```

or something

Reply:
ah, right, no need to expand... Done.